src/pages/Agregator/Agregator.types.ts

```diff
@@ -1,20 +1,21 @@
 import { BigNumber } from 'ethers';
 import { ChainEnum } from '../../config/chains';
 import { TokenEnum } from '../../config/tokens';
-import { AggregatorInputs } from './Agregator.fields';
+import { AgregatorInputs } from './Agregator.fields';
 
 export type AgregatorFormValues = {
-  [AggregatorInputs.ChainDropdown]: ChainEnum | '';
-  [AggregatorInputs.TokenDropdown]: TokenEnum | '';
-  [AggregatorInputs.SendAmount]: string;
-  [AggregatorInputs.DestinationChain]: ChainEnum;
-  [AggregatorInputs.ReceiveAmount]: string;
-  [AggregatorInputs.ReceiveAddress]: string;
+  [AgregatorInputs.StartingChain]: ChainEnum | '';
```

Comment:
Hmm, I'm afraid it may not work. This is due to the weird implementation of MuiSelect placeholder, which needs some value (cannot be undefined) https://github.com/mui/material-ui/issues/11069

Reply:
Hmm, but I think DefaultValues<AgregatorValues> should allow empty string

Reply:
There's one more thing about that, I forgot about it. When we set `defaultValue` to `undefined`, once the user chooses something, we get a MUI Error:
![Screenshot from 2022-03-04 17-07-30](https://user-images.githubusercontent.com/38423054/156798372-91a5fb98-c8f0-4346-a1c0-f706aed88387.png)